### ISSUE TITLE
North Dakota Addresses

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#2150]
North Dakota is just empty.  Lets fill in the gaps!
Day one - reached out to the following counties:Bottineau, Burke, Cavalier, Divide, Pembina, Renville, Rolette, and Towner.  I am very optimistic about Pembina (http://pembinacountynd.gov/county/departments/it-gis-and-maps/ ) but no other county has a gis website, so hopefully everyone I emailed can help us out somehow.  

Im going to email these in groups a row a day - total project time would be five days.  

day 1 progress:
![nd_day_1](https://cloud.githubusercontent.com/assets/5324316/20523031/3866c342-b081-11e6-83e3-ecc5427dae88.jpg)

**Comment by justinelliotmeyers** [comment]
Divide County Response:
Divide County does not currently have that information available. Our 911 data and dispatching is managed by ND State Radio; I am trying to get a GIS program in place here but resources are very limited.
 
JODY S. GUNLOCK
Emergency Services/911 Coordinator
 
701-965-6361
jgunlock@nd.gov

**Comment by justinelliotmeyers** [comment]
**
https://www.nd.gov/des/state-radio/telephone-services/
spoke to Mr. Phil Peterschick (ND Emergency Services)

they have a statewide address dataset - will speak to director and see if this is something we can obtain for the project.  

Stay tuned!

**Comment by justinelliotmeyers** [comment]
![nd_day_2](https://cloud.githubusercontent.com/assets/5324316/20625987/7e7dc6e0-b2e4-11e6-8299-9e3bd0b45af0.jpg)

day two - one county has open data in this batch!
also:

Ramsey has this: https://gis.co.ramsey.nd.us/arcgis/rest/services/Basemap/Parcels/MapServer/0
McHenry has this: http://services6.arcgis.com/zbSH183Elt4hJ5Hl/ArcGIS/rest/services/Parcels/FeatureServer/0
 But I cannot query them; any thoughts @iandees ?  Thanks!

**Comment by iandees** [comment]
I can query the McHenry one, but it looks like the gis.co.ramsey.nd.us server blocks the query URL somehow. We should hang on to the servers and keep trying later.

**Comment by justinelliotmeyers** [comment]
![nd_day_3](https://cloud.githubusercontent.com/assets/5324316/20709867/a541881a-b605-11e6-9dac-5d411b70cd35.jpg)

Finished contacting all counties in ND!!!!!  I am still waiting on Mr. Phil Peterschick (ND Emergency Services) to see if we can receive a state wide dump - he was super nice and seemed like he would share it, but had to ask.  He told me worst case scenario we would have to reach out to each county individually.

**Only two counties sent me data:
Pembina **no actual address data** - https://www.dropbox.com/sh/96kf02o11avjrtx/AAB7GXPuUyFqMM3yStmX2BTRa?dl=0

As per Pembina Counties Response with regard to my request for Address Points 
"Justin,
 
Unfortunately, Pembina County Government in North Dakota cannot fulfill your request. According to North Dakota Century Code 57-40.6-07, Published names and telephone numbers maintained by an emergency services communication system coordinator or public safety answering point are exempt records as defined in section 44-04-17.1. Address points fall under this exempt status, with or without identification of who might live there.
 
Best regards,
 
Liz
 
 Liz Beck | 911-IT-GIS Coordinator | Pembina County, ND | 701-265-8222 | lbeck@nd.gov
308 Court House Dr, #7, Cavalier, ND  58220 | www.pembinacountynd.gov"

However you can openly request and access the parcel data...!??

"Below is a link to Pembina County’s ftp site. The parcel shapefile is in the “parcel data” folder, and was updated 10/27/16. An updated shapefile is generally uploaded every two months. If the Owner field is empty, then the taxpayer is the owner. As I said in our phone conversation, parcel data is public record. My primary concern for attempting to come up with addresses based on parcel location is the method may result in inaccuracies. One example would be structures on any agriculture parcels may not be in the centroid of a given polygon. Another example is that addressing conventions, which are determined by each incorporated community, may be different from another. It would be required to know that convention in order to come up with accurate results."

So I do not think we are going to have address points for Pembina County...  Although their counties website seems like it is open, free, etc... http://pembinacountynd.gov/county/departments/it-gis-and-maps/



Mountrail **address data is in an excel file and gis is in an ESRI FGDB**- https://www.dropbox.com/sh/u2pp1z9aryx3cvs/AABWar2BjcO5qIJo_CYFeQO4a?dl=0

**Comment by justinelliotmeyers** [comment]
Another problem with North Dakota is that many of the counties are very rural.  This means there is no GIS department.  They hired a third party company to create and maintain their gis data - Sidwell - http://www.sidwellco.com/

I spoke to someone from Sidwell and they have different agreements with each county.  If you request the data from the county, they have Sidwell contact you and basically quote you a price...  Here are the rough prices: $.40 per parcel from 0-5,000 parcels $.35 per parcel from 5,000-10,000 and $.30 per parcel for any amount over 10,000.  The person from Sidwell said he would send me more information as per what counties they can sell data for - He said now the current number is 11 counties, and that those range from 3,000 parcels to some over 10,000.  If they do not have an agreement with the county, Sidwell told me we would have to contact individual counties to purchase the data or ask for permission to use the data for our project.  But it seems like we could buy it for the 11 counties and use it in OA - but I am waiting for and official response in an e-mail so this is documented.

**Comment by justinelliotmeyers** [comment]
Sent a follow-up e-mail to Mr. Phil Peterschick (ND Emergency Services) (has been seven days since I spoke to him).  Hopefully they have good news for us!?!

**Comment by justinelliotmeyers** [comment]
Mountrail Issue Created - https://github.com/openaddresses/openaddresses/issues/2169
Richland Issue Created - https://github.com/openaddresses/openaddresses/issues/2168

**Comment by justinelliotmeyers** [comment]
Foster County Response- 

We are not interested in participating at this time. Thank you! 
 
 
-J
 
Jess Earle
Foster County Emergency Management
Emergency Manager/911 Coordinator
1030 1st St N
Carrington, ND 58421
jdearle@nd.gov
701.652.2252 (Office)
701.650.2085 (Cell)

**Comment by nvkelso** [comment]
**Worst case fee:** 11 counties at 10,000 parcels each at $0.35 a parcel = `$38,500`.
**Medium case fee:** 11 counties at 4,500 parcels each at $0.40 a parcel = `$19,800`.

**Comment by justinelliotmeyers** [comment]
Don't kill the messenger!?!  I'm still optimistic we will get a state -wide dataset ;)

**Comment by nvkelso** [comment]
No worries! I’m mostly thinking about what a larger data acquisition budget
might look like.

**Comment by justinelliotmeyers** [comment]
emails are starting to come in and it is all over the place - but I believe the state runs the counties in brown here: 
![state-radio-service-area](https://cloud.githubusercontent.com/assets/5324316/20768492/48d45866-b70c-11e6-9fcf-108f94731533.jpg)

Haven't heard back from Phil yet.  If I don't by tomorrow I will call and touch base.

**Comment by justinelliotmeyers** [comment]
25 counties have the state maintain their data.  Just spoke to Phil on the phone - He has not had an opportunity to speak directly to his director.  They are dealing with DAPL and do not know when it can be discussed; but he will bring it up I was told.

**Comment by justinelliotmeyers** [comment]
Griggs county response: 
Hello Justin,
This is the response I got back from our 911 Coordinator. Please let me know if I can do anything else for you or you may email him as well.
Thanks, 
Samantha M Larson
Griggs County Auditor

---------- Forwarded message ----------
From: Wayne Oien <wayne.oien@griggscountynd.gov>
Date: Mon, Dec 12, 2016 at 2:19 PM
Subject: Re: Fwd: Griggs County, ND GIS Data Request
To: Samantha Quast <samantha.larson@griggscountynd.gov>


Hi Sam,
We don't have the data in a shape file, as far as I know. The State DES is working on developing this.
Wayne

**Comment by nvkelso** [comment]
@justinelliotmeyers Looks like issues per county that have data have been created? Is there any more followups here for this general issue or should we close it?

**Comment by justinelliotmeyers** [comment]
@nvkelso I called and spoke with Mr. Phil Peterschick (ND Emergency Services) today.  We talked for about thirty minutes.  He told me his director said no to releasing the data at this time.  He brought up the state statute   He is still optimistic about the data becoming open, which gives me hope...  But I don't know where that leaves us.  He told me there is a conference in March and would have more information then.  I asked if I could follow-up and he said please do.  Here is his e-mail from today:
Hi Justin,
Thanks again for your time today. As I mentioned here is a link that I referenced state century code in regarding the subscription to access the data.


http://www.legis.nd.gov/cencode/t37c17-3.pdf

Thanks,

Phil Peterschick, GISP
GIS Section Chief
ND Division of Homeland Security, Department of Emergency Services
701-328-8120

**Comment by justinelliotmeyers** [comment]
I have to create one more ticket for McLean County, ND - I have been back and forth with their E-911 mapping/ gis person their.  He sent me address data with no attributes - then some attributes...  Finally yesterday he sent all the address attributes he had - which is only for rural addresses (*no idea why urban are empty...).    
Justin,
You got the rural areas.  The GIS guy that helps is working on 911 computers right now.  You might not see any urban stuff until January sometime.
 
Todd A. Schreiner
McLean County
Equalization / Land Use / Veteran Service / E-911 Mapping / GIS
tschreiner@nd.gov

**Comment by justinelliotmeyers** [comment]
@nvkelso is it okay to create a ticket for just rural data for McLean County?

**Comment by nvkelso** [comment]
Yes, new issue just for rural part of McLean County.

**Comment by justinelliotmeyers** [comment]
McLean County Ticket here https://github.com/openaddresses/openaddresses/issues/2248

did not add PR because is dropbox link.  There might be one or two counties in the process of sending me data, but I will just create new tickets once I receive data.  I set a follow-up in my calendar to touch base with Mr. Petershick in March.  I also asked if we could purchase the data for a one time fee or subscription; he said he would speak with his director about it - but mentioned the issues with that and then having counties want $ instead of opening data - So I think we can close this ticket if that is okay @nvkelso ?  I can do a write up if needed.

**Comment by nvkelso** [comment]
Sounds good.

Please do small write up in this ticket to summarize findings. Then close; if new counties come thru make new per county
Issues.

**Comment by justinelliotmeyers** [comment]
Reaching out to all 53 counties in North Dakota was a bit of a challenge. Many of the counties are rural, don't have straight forward webpages, or any gis at all. After my first large push of finding data, I was informed that the state is putting together a state wide dataset. This was an amazing discovery; it would save eveyone so much time. After speaking to Phil Peterschick (the state gis person in charge) he seemed very optimistic, but told me that worse case scenario, I would have to reach out to each individual county. So I continued down my list. I set a timeframe up as five days, but completed in three. Most of the people who did respond were very helpful - many didn't have a gis/ gis data and set up an agreement with a third party company to create and maintain their gis data (this was typically Sidwell www.sidwellco.com ).  After reaching out to them directly, I found out it could be very expensive for us to purchase about 11 counties. So in final response from Mr. Peterschick, they decided against sharing the data at this time. He told me to follow-up in March after their e911 state meeting, and the answer may change then. I put it in my calendar.  So I believe in this journey, I found an additional five counties worth of data. I am still having e-mails trickle back in, and will continue to update new github tickets with data if received.
